### ISSUE TITLE
Add toggle to show/hide password to password fields

### ISSUE THREAD

**Issue by momijizukamori** [dreamwidth/dreamwidth#3037]
This is an accessibility feature, particularly on mobile where it's easier to mistype on software keyboards if you have issues with fine motor controls

Should be screenreader/keyboard accessible, and ideally shouldn't interfere with password manager extensions' fill options (personal pet peeve - I can provide LastPass and 1Password examples, probably)

Specific spots that need to be updated - https://github.com/dreamwidth/dreamwidth/blob/main/cgi-bin/DW/Template/Plugin/FormHTML.pm#L413 and possibly https://github.com/dreamwidth/dreamwidth/blob/main/cgi-bin/LJ/HTMLControls.pm#L351

**Comment by cmho** [comment]
claiming
